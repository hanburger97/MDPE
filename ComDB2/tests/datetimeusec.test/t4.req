delete from dtus where 1
set timezone UTC
insert into dtus(id, dt) values(1, 3600.246)
insert into dtus(id, dt) values(2, 3600.246246)
insert into dtus(id, dt) values(3, "7200.246")
insert into dtus(id, dt) values(4, cast("2015-02-23T080000.246246" as datetimeus))
insert into dtus(id, dt) values(5, "2015-02-23T080000.246")
insert into dtus(id, dt) values(6, "2015-02-23T080000.246246")
select id, dt from dtus order by dt, id
select id, dt from dtus where dt > 10000 order by id
set timezone US/Eastern
select id, dt from dtus order by dt, id
insert into dtus(id, ds) values(7, 3600.246)
insert into dtus(id, ds) values(8, 3600.246246)
insert into dtus(id, ds) values(9, "7200.246")
insert into dtus(id, ds) values(10, cast(1.246 as second))
insert into dtus(id, ds) values(11, cast(1.246246 as second))
insert into dtus(id, ds) values(12, 10)
insert into dtus(id, ds) values(13, "10 00:00:00.123456")
insert into dtus(id, ds) values(14, "0 02:04:06.246")
select id, ds from dtus order by ds, id
select id, ds from dtus where ds > "0 01:00:00" order by ds, id
select id, dt, ds from dtus order by id
