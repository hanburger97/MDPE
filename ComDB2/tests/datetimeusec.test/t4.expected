(rows deleted=14)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(id=1, dt="1970-01-01T010000.246 UTC")
(id=2, dt="1970-01-01T010000.246 UTC")
(id=3, dt="1970-01-01T020000.246 UTC")
(id=4, dt="2015-02-23T080000.246 UTC")
(id=5, dt="2015-02-23T080000.246 UTC")
(id=6, dt="2015-02-23T080000.246 UTC")
(id=4, dt="2015-02-23T080000.246 UTC")
(id=5, dt="2015-02-23T080000.246 UTC")
(id=6, dt="2015-02-23T080000.246 UTC")
(id=1, dt="1969-12-31T200000.246 US/Eastern")
(id=2, dt="1969-12-31T200000.246 US/Eastern")
(id=3, dt="1969-12-31T210000.246 US/Eastern")
(id=4, dt="2015-02-23T030000.246 US/Eastern")
(id=5, dt="2015-02-23T030000.246 US/Eastern")
(id=6, dt="2015-02-23T030000.246 US/Eastern")
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(id=1, ds=NULL)
(id=2, ds=NULL)
(id=3, ds=NULL)
(id=4, ds=NULL)
(id=5, ds=NULL)
(id=6, ds=NULL)
(id=10, ds="0 00:00:01.246")
(id=11, ds="0 00:00:01.246")
(id=12, ds="0 00:00:10.000")
(id=7, ds="0 01:00:00.246")
(id=8, ds="0 01:00:00.246")
(id=9, ds="0 02:00:00.246")
(id=14, ds="0 02:04:06.246")
(id=13, ds="10 00:00:00.123")
(id=7, ds="0 01:00:00.246")
(id=8, ds="0 01:00:00.246")
(id=9, ds="0 02:00:00.246")
(id=14, ds="0 02:04:06.246")
(id=13, ds="10 00:00:00.123")
(id=1, dt="1969-12-31T200000.246 US/Eastern", ds=NULL)
(id=2, dt="1969-12-31T200000.246 US/Eastern", ds=NULL)
(id=3, dt="1969-12-31T210000.246 US/Eastern", ds=NULL)
(id=4, dt="2015-02-23T030000.246 US/Eastern", ds=NULL)
(id=5, dt="2015-02-23T030000.246 US/Eastern", ds=NULL)
(id=6, dt="2015-02-23T030000.246 US/Eastern", ds=NULL)
(id=7, dt=NULL, ds="0 01:00:00.246")
(id=8, dt=NULL, ds="0 01:00:00.246")
(id=9, dt=NULL, ds="0 02:00:00.246")
(id=10, dt=NULL, ds="0 00:00:01.246")
(id=11, dt=NULL, ds="0 00:00:01.246")
(id=12, dt=NULL, ds="0 00:00:10.000")
(id=13, dt=NULL, ds="10 00:00:00.123")
(id=14, dt=NULL, ds="0 02:04:06.246")
